datacube_stats/output_drivers.py

```diff
@@ -560,15 +573,49 @@ def open_output_files(self):
 def _format_filename(path_template, **kwargs):
```

Comment:
This function doesn't appear to be used. Maybe this was an already existing mistake though.